### PR TITLE
documentation: update development plan

### PR DESCRIPTION
**Pull request by fondsonlu**
Co-authored-by: 
@paytonchan 
@chene40 
@angelaw7 
@tan-jason

### CODE REVIEW

**Comment by chene40** [comment]
PR closes #6

File: docs/DevelopmentPlan/DevelopmentPlan.tex [hunk 1]
```diff
@@ -1,240 +1,396 @@
...
+\end{itemize}
 
-\wss{How will the meetings be structured?  There should be a chair for all meetings.  There should be an agenda for all meetings.}
+\newpage
 
 \section{Team Communication Plan}
 
-\wss{Issues on GitHub should be part of your communication plan.}
+\begin{itemize}
+    \item The main source of communication between the team will be via \textbf{Discord}. Meetings that need to be conducted virtually will be done through the dedicated voice channel (\textit{General}) within the team server.
```
Comment (tan-jason) [inline docs/DevelopmentPlan/DevelopmentPlan.tex:50]: NIT: do we have to specify the server channel name?

File: docs/DevelopmentPlan/DevelopmentPlan.tex [hunk 2]
```diff
@@ -1,240 +1,396 @@
...
 \end{itemize}
 
-\section{Project Decomposition and Scheduling}
-
+\subsection{GitHub Issues}
 \begin{itemize}
-  \item How will you be using GitHub projects?
-  \item Include a link to your GitHub project
+    \item will be used to track and manage all development tasks.
+    \begin{itemize}
```
Comment (tan-jason) [inline docs/DevelopmentPlan/DevelopmentPlan.tex:111]: not sure if we wanna specify how the work will be divided up? (i.e. horizontal slicing vs. vertical)

File: docs/DevelopmentPlan/DevelopmentPlan.tex [hunk 3]
```diff
@@ -1,240 +1,396 @@
...
+            \item \texttt{chore:} - for project related commits such as bumping project/dependency versions, file renames/relocations, etc.
+        \end{itemize}
+        \item Example commit messages:
+        \begin{itemize}
+            \item \texttt{feat: generate openapi ts client based on api\_spec.yaml}
+            \item \texttt{fix: change color of CTA button from red to blue}
+            \item \texttt{chore: reorganize file structure hierarchy}
+        \end{itemize}
+    \end{itemize}
+    \item We will use PEP 8 style guides for Python code
```
Comment (tan-jason) [inline docs/DevelopmentPlan/DevelopmentPlan.tex:393]: NIT: not sure if this is too much for this doc

File: docs/DevelopmentPlan/DevelopmentPlan.tex (hunk 1 above)
Comment (chene40) [reply docs/DevelopmentPlan/DevelopmentPlan.tex:50]: dont think so

File: docs/DevelopmentPlan/DevelopmentPlan.tex [hunk 4]
```diff
@@ -1,240 +1,396 @@
...
-
-\newpage{}
-
-\wss{Put your introductory blurb here.  Often the blurb is a brief roadmap of
-what is contained in the report.}
-
-\wss{Additional information on the development plan can be found in the
-\href{https://gitlab.cas.mcmaster.ca/courses/capstone/-/blob/main/Lectures/L02b_POCAndDevPlan/POCAndDevPlan.pdf?ref_type=heads}
-{lecture slides}.}
-
```
Comment (angelaw7) [inline docs/DevelopmentPlan/DevelopmentPlan.tex:40]: add these back D:

Comment (chene40) [reply docs/DevelopmentPlan/DevelopmentPlan.tex:40]: @fondsonlu choobydooki what happened DDDDDD; stop nuking our intros

**Comment by chene40** [comment]
my brodi is making steak now lol he says to just make a separate PR and add our blurbs in if we want @angelaw7

File: docs/DevelopmentPlan/DevelopmentPlan.tex (hunk 4 above)
Comment (angelaw7) [reply docs/DevelopmentPlan/DevelopmentPlan.tex]: added this back

**Comment by chene40** [comment]
MERGE THAT JAJAJAJA

**Pull request merged**